src/message_bus.zig

```diff
@@ -996,7 +996,7 @@ fn MessageBusImpl(comptime process_type: ProcessType) type {
                 assert(self.peer == .client or self.peer == .replica);
                 assert(self.state == .connected);
                 assert(self.fd != -1);
-                const message = self.send_queue.peek() orelse return;
+                const message = (self.send_queue.peek() orelse return).*;
```

Comment:
Here we do have to dereference the result as `peek()` returns a `**Message`, so this change is necessary albeit a little ugly.